podpac/core/coordinates/coordinates.py

```diff
@@ -1501,12 +1504,77 @@ def issubset(self, other):
 
         return all(c.issubset(other) for c in self.values())
 
-    def is_stacked(self, dim):
-        if dim not in self.udims:
+    def is_stacked(self, dim):  # re-wrote to be able to iterate through c.dims

```

Comment:
Is there a reason you're not using the individual coordinate's `is_stacked` method?